ganalytics.cpp

```diff
@@ -856,8 +868,21 @@ void GAnalytics::Private::postMessage()
         networkManager = new QNetworkAccessManager(this);
     }
 
-    QNetworkReply *reply = networkManager->post(request, ba);
-    connect(reply, SIGNAL(finished()), this, SLOT(postMessageFinished()));
+    if (onlinePosting)
+    {
+        QNetworkReply *reply = networkManager->post(request, ba);
+        connect(reply, SIGNAL(finished()), this, SLOT(postMessageFinished()));
+    }
+    else
+    {
+        qDebug() << "[Analytics message]";
+        qDebug() << ba;
```

Comment:
Great, but could we probably change the debug to use the short version:
 - start with "GANALYTICS: " (without newline)
 - show [query arguments](https://doc.qt.io/qt-5/qurlquery.html#queryItems) as `key = value`, separated via `; `
 - show rawHeader `User-Agent` only
 - show this as one-line report.